### PR TITLE
Change NPC sneak attack entries to new rules

### PR DESCRIPTION
**Pull request by TikaelSol**
Also did some brushing up while I was at it. The Norgorberite Spy in SoT will need gang up automation added when this is merged.

### CODE REVIEW

File: packs/fists-of-the-ruby-phoenix-bestiary/butterfly-blade-warrior.json [hunk 1]
```diff
@@ -390,7 +390,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The warrior's Strikes deal an additional [[/r 1d6]]{1d6 precision damage} to off-guard creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>The warrior's Strikes deal an additional [[/r 1d6]]{1d6 precision damage} to off-guard creatures.</p>"
```
[the remark this answers is not in the document]
Comment (TikaelSol) [reply packs/fists-of-the-ruby-phoenix-bestiary/butterfly-blade-warrior.json:393]: Yeah, we should fix those. Thought I caught them but the label must have caused it to slip by unnoticed.

File: packs/abomination-vaults-bestiary/drow-warden.json [hunk 2]
```diff
@@ -1315,7 +1315,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>A warden deals an extra 1d6 damage to off-guard creatures.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>A warden deals an extra 1d6 damage to off-guard creatures.</p>"
```
Comment (simonward) [inline packs/abomination-vaults-bestiary/drow-warden.json:1318]:
```suggestion
                    "value": "<p>A warden deals an extra 1d6 damage to @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] creatures.</p>"
```

**Comment by simonward** [comment]
The kobold dragon mages all have: "The dragon mage deals an extra [[/r {1d6}]]{1d6 damage} to creatures"

File: packs/troubles-in-otari-bestiary/omblin-leadbuster.json [hunk 3]
```diff
@@ -589,7 +589,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>Omblin deals an extra 1d6 to creatures that have the off-guard condition.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>Omblin deals an extra 1d6 to creatures that have the off-guard condition.</p>"
```
Comment (simonward) [inline packs/troubles-in-otari-bestiary/omblin-leadbuster.json:592]:
```suggestion
                    "value": "<p>Omblin deals an extra 1d6 to creatures that have the @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] condition.</p>"
```

File: packs/troubles-in-otari-bestiary/kobold-trapmaster.json [hunk 4]
```diff
@@ -625,7 +625,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The kobold trapmaster deals an extra 1d6 damage to creatures that have the off-guard condition.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>The kobold trapmaster deals an extra 1d6 damage to creatures that have the off-guard condition.</p>"
```
Comment (simonward) [inline packs/troubles-in-otari-bestiary/kobold-trapmaster.json:628]:
```suggestion
                    "value": "<p>The kobold trapmaster deals an extra 1d6 damage to creatures that have the @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] condition.</p>"
```

File: packs/strength-of-thousands-bestiary/norgorberite-spy.json [hunk 5]
```diff
@@ -668,7 +668,7 @@
                 },
                 "category": "offensive",
                 "description": {
-                    "value": "<p>The Norgorberite spy deals an additional 4d6 precision damage to off-guard creatures or creatures within reach of at least two of their allies.</p>\n<hr />\n<p>@Localize[PF2E.NPC.Abilities.Glossary.SneakAttack]</p>"
+                    "value": "<p>The Norgorberite spy deals an additional 4d6 precision damage to @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] creatures. or creatures within reach of at least two of their allies.</p>"
```
Comment (simonward) [inline packs/strength-of-thousands-bestiary/norgorberite-spy.json:671]:
```suggestion
                    "value": "<p>The Norgorberite spy deals an additional 4d6 precision damage to @UUID[Compendium.pf2e.conditionitems.Item.Off-Guard] creatures or creatures within reach of at least two of their allies.</p>"
```

**Comment by simonward** [comment]
Looks like several of the bb NPCs have the same pattern:
The drow sneak deals an extra [[/r {1d6}]]{1d6 damage} to creatures
Zolgran deals an extra [[/r {1d6}]]{1d6 damage} to creatures
The kobold scout deals an extra [[/r {1d6}]]{1d6 damage} to creatures
The kobold trapmaster deals an extra [[/r {1d6}]]{1d6 damage} to creatures
The leopard deals an extra [[/r {1d4}]]{1d4 damage} to creatures

File: packs/strength-of-thousands-bestiary/norgorberite-spy.json
Comment (simonward) [inline packs/strength-of-thousands-bestiary/norgorberite-spy.json:1]: The Norgorberite Spy still has:
`{
	"or": [
		"finesse",
		"agile",
		"ranged"
	]
}`
but I assume that's part of your comment about it needing gang up automation?

**Review by simonward — APPROVED** [review]

File: packs/strength-of-thousands-bestiary/norgorberite-spy.json
Comment (TikaelSol) [reply packs/strength-of-thousands-bestiary/norgorberite-spy.json:1]: yes, that one and Doatara the poisoner are on my list to do as follow ups

**Review by SpartanCPA — APPROVED** [review]

**Pull request merged**